From: Andreas Gruenbacher <agruen@suse.de>
Subject: Add missing EXPORT_SYMBOL(strnlen_user_{skas,tt})

The strnlen_user function which these functions implement for uml
are exported as well.

Index: linux-2.6.16/arch/um/kernel/skas/uaccess.c
===================================================================
--- linux-2.6.16.orig/arch/um/kernel/skas/uaccess.c
+++ linux-2.6.16/arch/um/kernel/skas/uaccess.c
@@ -249,6 +249,7 @@ int strnlen_user_skas(const void __user 
 		return(count + 1);
 	return(-EFAULT);
 }
+EXPORT_SYMBOL(strnlen_user_skas);
 
 /*
  * Overrides for Emacs so that we follow Linus's tabbing style.
Index: linux-2.6.16/arch/um/kernel/tt/uaccess.c
===================================================================
--- linux-2.6.16.orig/arch/um/kernel/tt/uaccess.c
+++ linux-2.6.16/arch/um/kernel/tt/uaccess.c
@@ -60,6 +60,7 @@ int strnlen_user_tt(const void __user *s
 				 &current->thread.fault_addr,
 				 &current->thread.fault_catcher));
 }
+EXPORT_SYMBOL(strnlen_user_tt);
 
 /*
  * Overrides for Emacs so that we follow Linus's tabbing style.
